### ISSUE TITLE
Server-timing headers

### ISSUE THREAD

**Issue by clarfonthey** [go-gitea/gitea#16747]
It would be nice if we could enable the option for gitea to serve [`Server-Timing`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing) headers with responses, as it would be very helpful with debugging.

The option `SHOW_FOOTER_TEMPLATE_LOAD_TIME` should be incorporated into this instead, as more data can be included with the timing and it doesn't add information in user-visible areas.

**Comment by zeripath** [comment]
The only way a Server-Timing header can give the correct value is if the all of the content is pre-rendered before writing to the response. I don't think that's guaranteed.

**Comment by clarfonthey** [comment]
True, then I guess that template rendering time should be a separate option. But we can still separate DB latency, etc. in the header.
